src/libraries/System.Diagnostics.Process/src/System/Diagnostics/ThreadState.cs

```diff
@@ -3,49 +3,37 @@
 
 namespace System.Diagnostics
 {
-    /// <devdoc>
-    ///     Specifies the execution state of a thread.
-    /// </devdoc>
+    /// <summary>Specifies the current execution state of the thread.</summary>
+    /// <remarks><format type="text/markdown"><![CDATA[
+    /// > [!IMPORTANT]
```

Comment:
I haven't kept up with discussion -- is this `<![CDATA[[!IMPORTANT]` style what we will use log term? Why not just `<important>` or something?

also by using CDATA, the internal tags are not validated (eg the `xref` tag here).

Reply:
The consensus was that whenever the tool finds items that have no way of getting converted from markdown to xml, we will preserve the CDATA section, to avoid losing data.

For example: `[name](url)` or `[!INCLUDE[name](~/path/to/file)]` have no xml equivalent.

These are the only xml tags we support within a remarks section:
![image](https://user-images.githubusercontent.com/1175054/108793260-fa6c8880-7537-11eb-8cc1-930aeb3063dc.png)

You're correct, xrefs don't get validated in source, but consider that this is the backport of what we already had in MS Docs, so it's pre-validated. I expect devs to add future new remarks as xml comments.